hoomd/__init__.py

```diff
@@ -87,4 +87,5 @@ def _hoomd_sys_excepthook(type, value, traceback):
 
 sys.excepthook = _hoomd_sys_excepthook
 
-__version__ = "{0}.{1}.{2}".format(*_hoomd.__version__)
+#  __version__ = "{0}.{1}.{2}".format(*_hoomd.__version__)
+__version__ = "3.0.0"
```

Comment:
Set this to 2.9.0 and add `__init__.py` to the bumpversion configuration in `setup.cfg`. Then bumpversion will keep it up to date with the others.

Reply:
👍 done. I just wanted to confirm that you were fine with maintaining the version separately in Python.